content/10.diagnostics.md

```diff
@@ -350,13 +356,13 @@ IgG titers peaked by day 60, and persisted in all donors through the two-year du
 Such tests can also illuminate the progression of viral disease, as IgM are the first antibodies produced by the body and indicate that the infection is active.
 Once the body has responded to the infection, IgG are produced and gradually replace IgM, indicating that the body has developed immunogenic memory [@doi:10.1002/jmv.25820].
 Therefore, it was hoped that the development of assays to detect the presence of IgM and IgG antibodies against SARS-CoV-2 would allow the identification of cases from early in the infection course (IgM) and for months or years afterwards (IgG).
-This information can be useful for understanding the extent to which a viral threat has affected a particular population.
 Several technologies have been used to developed serological tests for COVID-19, including ELISA, lateral flow immunoassay, chemiluminescence immunoassay, and neutralizing antibody assays [@doi:10.1093/ajcp/aqaa112].
 
 #### ELISA
 
-The application of ELISA to serological testing is complementary to its use in molecular diagnostics (above).
-Instead of using an enzyme-labeled antibody as a probe that binds to the target antigen, instead the probe is an antigen and the target is an antibody.
+The application of ELISA to serological testing is complementary to its use in molecular diagnostics (described above).
+Instead of using an enzyme-labeled antibody as a probe that binds to the target antigen, the probe is an antigen and the target is an antibody.
+The enzyme used for detection and signal amplification is on a secondary antibody raised generally against human IgG or IgM.<!--Request review on this sentence-->
```

Comment:
@cbrueffer @nilswellhausen @RLordan Is anyone able to review this sentence? (review requested by the author, @jessegmeyerlab)

Reply:
I think this is correct + anti-human pan Ig as per: https://doi.org/10.1101/2020.04.24.057323